src/dnlib.csproj

```diff
@@ -15,6 +15,8 @@
     <FileAlignment>512</FileAlignment>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\dnlib.snk</AssemblyOriginatorKeyFile>
+    <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+    <FrameworkPathOverride Condition="'$(OS)' != 'Windows_NT'">/usr/lib/mono/2.0-api/</FrameworkPathOverride>
```

Comment:
This is a hard coded path. Is it not possible for the build tools to find this path themselves?